src/rust/ensogl/src/display/shape/text/text_field/cursor.rs

```diff
@@ -658,7 +646,7 @@ mod test {
         let properties     = TextFieldProperties::default(&mut fonts);
         let mut content    = TextFieldContent::new(text,&properties);
         let mut navigation = CursorNavigation
-            {selecting:true, ..CursorNavigation::default(&mut content)};
+            {selecting:true, ..CursorNavigation::new(&mut content)};
```

Comment:
I doubt it compiles now. Please check tests before requesting for review.

Reply:
Fixed!